share/spice/symbolab/symbolab.js

```diff
@@ -0,0 +1,48 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_symbolab = function (api_result) {
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || !api_result.solution) {
+            return Spice.failed('symbolab');
+        }
+
+        DDG.require('mathquill', function (){
+            Spice.add({
+                id: "symbolab",
+                name: "Calculator",
+                data: api_result.solution,
+                meta: {
+                    sourceName: "Symbolab.com",
+                    sourceUrl: api_result.solution.url
+                },
+                normalize: function (data) {
+                    return {
+                        latex: data.originalQuery.replace(/ /g, "\\text{ }"),
+                        solution: data.solutionLatex.replace(/^=/, "").replace(/,/, ",\\text{ }"),
```

Comment:
Should we be replacing all of the `,`'s? If so, we need to add `/g` to the regex.

What exactly is this for? Perhaps you should add a comment so others will know as well